src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  background-color: #000;
+}
+
+.outer {
+  width: 348px;
+  margin: 20px 0 40px;
+  background-color: #008dc3;
+}
+
+.inner {
+  margin: 10px;
+  background-color: orange;
+}
+
+p {
```

Comment:
better to add class and add styles to it

Reply:
Do you like it now)?